tsconfig.build.json

```diff
@@ -1,12 +1,14 @@
 {
-  "extends": "./node_modules/@sentry/typescript/tsconfig.json",
+  "extends": "@sentry/typescript/tsconfig.json",
   "compilerOptions": {
     "target": "es2017",
     "baseUrl": ".",
     "outDir": "dist",
     "rootDir": "src",
     "types": ["node"],
-    "noImplicitThis": false
+    "noImplicitThis": false,
+    "esModuleInterop": true,
+    "importHelpers": true
```

Comment:
Are these somehow related to the import changes I'm seeing throughout the PR? Are these part-and-parcel of the task at hand or incidental?

Reply:
These are part-and-parcel of the task at hand as without these changes, `esbuild` cannot build the package. The reason being TypeScripts's legacy `import` behavior before ES6 module standard was established.